### PR TITLE
feat: modify charts and tests for new UI release

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
/deploy #persist

### CODE REVIEW

**Comment by olevski** [comment]
@lorenzo-cavazzi can you isolate the fix to the acceptance tests from the ui chart update? Because it seems to me that this PR tries to do both in one. But perhaps I am confused.

**Comment by lorenzo-cavazzi** [comment]
Sorry for the mess, this includes the first 3 commits from here https://github.com/SwissDataScienceCenter/renku/pull/2643 and the others are the fixes -- but it works only with the latest (yet unreleased) UI version.
I'll try to clean up tomorrow if all the tests pass (both here and in the UI PRs :crossed_fingers: )

**Comment by olevski** [comment]
@lorenzo-cavazzi no problem. I just wanted to make sure it wasn't accidental. You dont have to bother cleaning up. We can just merge this after the other PR you mentioned gets merged.

**Review by olevski — DISMISSED** [review]

**Review by olevski — APPROVED** [review]

File: scripts/generate-values/base-renku-values.yaml.template
Comment (rokroskar) [inline scripts/generate-values/base-renku-values.yaml.template:95]: I don't see this set in the default values in the renku chart - why is it set here? Seems like this default should either be set on the component chart or at least on the renku chart.

Comment (rokroskar) [reply scripts/generate-values/base-renku-values.yaml.template:95]: But more importantly, can't this always be built from the information contained in the `ui.server.serverData` values? Since the two charts are now unified, you shouldn't need to declare this manually.

Comment (lorenzo-cavazzi) [reply scripts/generate-values/base-renku-values.yaml.template:95]: Do you mean we should add it (commented) to `helm-chart/renku/values.yaml` ? Or we should drop it since it's set as a default in the UI values? If it's the latter, then we should probably drop  `baseUrl` and `gatewayUrl` too since the default is already derived from `ui.protocol` and `.Values.global.renku.domain`.

Comment (lorenzo-cavazzi) [reply scripts/generate-values/base-renku-values.yaml.template:95]: > can't this always be built from the information contained in the ui.server.serverData values?
Yes, we can unify some of the values from `client` and `server` but that requires a bit more work since the URLs are built in different ways and it would make sense to unify the logic -- should I make an issue in the UI to take care of that?

Comment (rokroskar) [reply scripts/generate-values/base-renku-values.yaml.template:95]: yes please do, there is certainly an opportunity here to reduce redundancy

**Review by pameladelgado — DISMISSED** [review]
🚀

File: scripts/generate-values/base-renku-values.yaml.template
Comment (rokroskar) [reply scripts/generate-values/base-renku-values.yaml.template:95]: drop as much as possible from this template (but make sure it still yields a usable values file) because it's meant as a first exposure for someone deploying renku

Comment (lorenzo-cavazzi) [reply scripts/generate-values/base-renku-values.yaml.template]: This is the issue in the UI https://github.com/SwissDataScienceCenter/renku-ui/issues/1976

I removed all the values except the `welcomePage` since they are not strictly needed.

**Review by rokroskar — APPROVED** [review]

**Pull request merged**
